package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "_electro-grammar",
+  "devDependencies": {
+    "babel-cli": "^6.24.1",
+    "babel-plugin-transform-object-assign": "^6.22.0",
+    "babel-preset-es2015": "^6.24.1",
+    "js-yaml": "^3.9.1"
+  },
+  "babel": {
+    "presets": [
+      "es2015"
+    ],
+    "plugins": [
+      "transform-object-assign"
+    ]
+  }
+}
```

Comment:
We already have a package.json for the js bit. I guess you want one for the CPL script, it doesn't need all these deps then though.